packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderThemeStyleTag.ts

```diff
@@ -1,63 +1,67 @@
-import { useId, usePrevious } from '@fluentui/react-utilities';
+import { useId, useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
 import * as React from 'react';
 import type { FluentProviderState } from './FluentProvider.types';
 import { fluentProviderClassNames } from './useFluentProviderStyles';
 
+const useInsertionEffect = (React as never)['useInsertion' + 'Effect']
```

Comment:
```suggestion
// String concatenation is used to prevent bundlers to complain with older versions of React
const useInsertionEffect = (React as never)['useInsertion' + 'Effect']
```

Let's put a comment so people will be aware :)